ui/src/app/cohort-search/attributes-page-v2/attributes-page-v2.component.tsx

```diff
@@ -432,7 +432,7 @@ export const AttributesPageV2 = fp.flow(withCurrentWorkspace(), withCurrentCohor
 
     validateForm() {
       const {form, isCOPESurvey} = this.state;
-      if (form.anyValue && (!isCOPESurvey || form.anyVersion)) {
+      if ((form.anyValue || form.num.length === 0) && (!isCOPESurvey || form.anyVersion)) {
```

Comment:
Question: can we add COVID attribute without selecting any version?

Reply:
If you don't select anything or enter any values (blank form) it treats it like any version/any value.